src/js/models/appUser.js

```diff
@@ -13,7 +13,9 @@ var AppUser = module.exports = Backbone.AssociatedModel.extend({
     },
 
     parse: function(data) {
-        return data.appUser;
+        var appUser = data.appUser;
+        appUser.signedUpAt = new Date(data.appUser.signedUpAt);
```

Comment:
So, this takes care of parsing the response from the server, but what takes care of serializing `signedUpAt` when we make a request to the server? Or does the server accept a ISO 8601 string?